log_config.yaml

```
@@ -12,7 +12,15 @@ handlers:
         level: DEBUG
         formatter: default
         stream: ext://sys.stderr
+    file_rotating:
+        class: logging.handlers.TimedRotatingFileHandler
+        level: DEBUG
+        filename: logs/ictrl_log.log
+        formatter: default
+        when: 'midnight'
```

Let's keep the style consistent, since the other string values are not wrapped
```suggestion
        when: midnight
```